recipes/perl-business-issn/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set perl_name = "Business::ISSN" %}
+{% set name = "perl-" + perl_name.replace("::", "-").lower() %}
+{% set author = "BDFOY" %}
+{% set version = "1.005" %}
+{% set sha256 = "3b09b0267f0a6660fb92b6f50c4c7796ef6a263b62ad3bbeaa07189a0c7c39b3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/{{ author[0] }}/{{ author[0:2] }}/{{ author }}/{{ perl_name.replace("::", "-") }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: |
+    perl Makefile.PL INSTALLDIRS=vendor
+    make
+    make test
+    make install
+  skip: true  # [win]
```

```suggestion
  noarch: generic
```
Any Perl recipe that doesn't use a native compiler should be `noarch: generic` and should not contain any `skip:` lines.  If Windows failures it can be ignored, because in the feedstock a `noarch: generic` package will only be built on `linux-64`.